.github/workflows/dependabot-automerge.yml

```
@@ -30,61 +30,87 @@ jobs:
+              --connect-timeout 10 \
+              --max-time 60 \
+              --retry 3 \
+              --retry-delay 2 \
+              --retry-all-errors \
+              -H "Authorization: bearer ${ACTIONS_ID_TOKEN_REQUEST_TOKEN}" \
+              "${ACTIONS_ID_TOKEN_REQUEST_URL}&audience=github"
+          )"
+
+          read -r repo ref <<<"$(
+            OIDC_JSON="${oidc_json}" python3 - <<'PY'
+import base64, json, os
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)